pkg/snowflake/operator.go

```diff
@@ -140,6 +141,11 @@ func (o *QuerySensor) RunTask(ctx context.Context, p *pipeline.Pipeline, t *pipe
 		return err
 	}
 
+	err = conn.CreateDataBaseIfNotExist(ctx, t)
+	if err != nil {
+		return err
+	}
```

Comment:
I suggest doing this only if the asset name is two parts with `<schema>.<table>`